udata/settings.py

```diff
@@ -37,6 +37,7 @@ class Defaults(object):
 
     SECURITY_PASSWORD_HASH = b'bcrypt'
     SECURITY_PASSWORD_SALT = b'udata'
+    SECURITY_REGISTERABLE = True
```

Comment:
Or maybe just put a comment on top of `SECURITY_*` to explain that it comes from flask-security?